draft-ietf-quic-transport.md

```diff
@@ -801,33 +801,37 @@ limit is increased.
 
 ## Flow Credit Increments {#fc-credit}
```

Comment:
How about "Managing Flow Control"?

Reply:
That seems too general. I'm not in love with what we have, but I prefer it because it's more specific.

Reply:
I can see that.  These flow control sections are a bit too long for my liking, but it's good enough for now.